test/apps/mapbox-integration/mapbox-layers-react/mapbox-layers-react.jsx

```diff
@@ -83,7 +83,7 @@ function App() {
       initialViewState={INITIAL_VIEW_STATE}
       controller={true}
       onWebGLInitialized={setGLContext}
-      glOptions={{stencil: true}}
+      deviceProps={{type: 'webgl', stencil: true}}
```

Comment:
User's can forget to add `type: 'webgl'`. Any problem with deck adding it by default if the user hasn't specified? Also technically if the user provides `type` here it's a luma `CreateDeviceProps` instead of `DeviceProps`